apps/demos/Demos/Chat/Customization/Angular/app/app.service.ts

```diff
@@ -0,0 +1,87 @@
+import { Injectable } from '@angular/core';
+import { Observable, BehaviorSubject } from 'rxjs';
+import { User, Message, MessageEnteredEvent } from 'devextreme/ui/chat';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AppService {
+  date: Date;
+
+  currentUser: User = {
+    id: 'c94c0e76-fb49-4b9b-8f07-9f93ed93b4f3',
+    name: 'John Doe',
+  };
+
+  supportAgent: User = {
+    id: 'd16d1a4c-5c67-4e20-b70e-2991c22747c3',
+    name: 'Support Agent',
+    avatarUrl: '../../../../images/petersmith.png',
```

Comment:
Now the image is not loaded
```suggestion
    avatarUrl: 'images/petersmith.png',
```

Reply:
[37c8016](https://github.com/DevExpress/DevExtreme/pull/28482/commits/37c8016387efd0b5991f5f9cb043fba1c686bd51)